app/models/doi.rb

```diff
@@ -5,7 +5,7 @@
 
 class Doi < ApplicationRecord
   PUBLISHER_JSON_SCHEMA = "#{Rails.root}/app/models/schemas/doi/publisher.json"
-  audited only: %i[doi url creators contributors titles publisher publisher_obj publication_year types descriptions container sizes formats version_info language dates identifiers related_identifiers related_items funding_references geo_locations rights_list subjects schema_version content_url landing_page aasm_state source reason]
```

Comment:
Will removing this remove historical data about changes to the field from the `activities` table/index/endpoint?

Reply:
Great question.  Deleting the field name from that line just tells the audit module (activities) to not record changes for that field.  It won't erase historical data for publisher or leave those changes out of a call to /activities. 

Changes are kept in the audited_changes column of the audits table.  For reporting changes, activities just grabs audited_changes, but doesn't care what is in it. For recording changes, the publisher column will be gone.